src/main/java/de/qabel/desktop/ui/contact/ContactController.java

```diff
@@ -274,8 +213,18 @@ private void buildGson() throws EntityNotFoundException, PersistenceException {
     public void update() {
         Platform.runLater(this::loadContacts);
     }
-}
-
 
+    public void openContactMenu(MouseEvent event) {
+        initializeMenu();
+        Platform.runLater(() -> contactMenuController.showMenu(event.getSceneX()
+            + layoutWindow.getScene().getWindow().getX(), event.getSceneY()
+            + layoutWindow.getScene().getWindow().getY() + contactMenu.getHeight()));
+    }
 
+    private void initializeMenu() {
+        contactMenuView = new ContactMenuView();
+        contactMenuView.getView(layoutWindow.getChildren()::add);
```

Comment:
this view is not added in the layoutWindow anymore. 

 private void initializeMenu(double coordX, double coordY) {
        contactMenuView = new ContactMenuView();
        contactMenuView.getView(view -> {
            popOver = new PopOver();
           ...
        });